providers/implementations/keymgmt/ml_kem_kmgmt.c

```diff
@@ -125,11 +127,19 @@ static int ml_kem_pairwise_test(const ML_KEM_KEY *key)
 }
 #endif  /* FIPS_MODULE */
 
-static void *ml_kem_new(OSSL_LIB_CTX *libctx, char *propq, int evp_type)
+static void *ml_kem_new(void *vctx, int evp_type)
```

Comment:
Wouldn't this be better using the correct type:
```suggestion
static void *ml_kem_new(PROV_CTX *vctx, int evp_type)
```